CityPuzzle/Login And SingUp/SignUpPage.xaml.cs

```diff
@@ -132,21 +106,18 @@ void Registration_Click(object sender, EventArgs e)
         async void ErrorAllert(string message)
         {
             await DisplayAlert("Klaida", message, "Gerai");
-
         }
 
-        private User CreateUser(string userName, string name, string lastName, string password,string email, double maxDist = 3)
+        private User CreateUser(string userName, string name, string lastName, string email, string password, double maxDist = 3)
         {
-            List<Lazy<Puzzle>> zero = new List<Lazy<Puzzle>>();
             User user = new User()
             {
                 Name = name,
                 LastName = lastName,
                 UserName = userName,
-                Pass = password,
                 Email = email,
-                MaxQuestDistance = maxDist,
-                QuestsCompleted = zero
```

Comment:
Why is QuestsCompleted deleted? Are you sure it's not needed?

Reply:
User class instantiates QuestsCompleted list by default on creation

Reply:
Checked it out, everything is fine.